test_conformance/images/kernel_read_write/test_cl_ext_image_from_buffer.cpp

```diff
@@ -629,7 +629,7 @@ int image_from_small_buffer_negative(cl_device_id device, cl_context context,
 
                 clCreateImage(context, flag, &format, &image_desc, nullptr,
                               &err);
-                test_failure_error(err, CL_INVALID_MEM_OBJECT,
+                test_failure_error(err, CL_INVALID_IMAGE_FORMAT_DESCRIPTOR,
```

Comment:
Should we accept `CL_IMAGE_SIZE` here also, or fix the spec to disambiguate which error code should be returned?

FWIW, if I understand this scenario correctly and I had to pick, I'd probably return `CL_IMAGE_SIZE` due to:

> an image is created from a buffer and the buffer passed in mem_object is too small